tests/integrations/android_sdk/AndroidSDK/test_verify_emulator.py

```diff
@@ -1,16 +1,29 @@
 import os
 import subprocess
+import sys
 
 import pytest
 
 from briefcase.exceptions import BriefcaseCommandError
 
+from ....utils import create_file
+
+
+def create_emulator(root_path):
+    # Create `emulator` within `root_path`.
+    if sys.platform == "win32":
+        create_file(
+            root_path / "emulator" / "emulator.exe", "The Emulator", chmod=0o755
+        )
+    else:
+        create_file(root_path / "emulator" / "emulator", "The Emulator", chmod=0o755)
```

Comment:
As the only difference betwen the two branches is in the filename extension, it would be clearer to factor the rest out.